### ISSUE TITLE
Add ability to import users

### ISSUE THREAD

**Issue by Mashenka** [wp-crm/wp-crm#35]
We have option in order to export users, but do not have option to import them to some particular site with wp-crm

**Comment by feratechinc** [comment]
Yep, can't use this CRM without the ability to import. I have 3 sites with 200+ users each... Not going to be inputting by hand. Sad, this looked promising but this single feature killed it for me.
